source/renderer/app/i18n/locales/en-US.json

```diff
@@ -376,4 +374,4 @@
   "wallet.transactions.no.transactions": "No transactions",
   "wallet.transactions.no.transactions.found": "No transactions found",
   "wallet.transactions.search.hint": "!!!Search transaction"
-}
+}
```

Comment:
why no newline at end of file?

Reply:
That’s automatic output od the translation manager...